website/docs/concepts/router.mdx

```diff
@@ -425,8 +433,6 @@ enum MainRoute {
     News,
     #[at("/contact")]
     Contact,
-    #[at("/settings/:s")]
-    Settings,
```

Comment:
I think `Settings { s: String }` would fix it?

Reply:
What do you mean?
There was no bug in the actual code before, but users kept trying to add `/settings` and that does not get matched, its a bug.

Reply:
I thought `route_recognizer` can match wildcard optionally...

In this case you can fix it with:

```rust
#[at("/settings/")]
SettingsHome,
#[at("/settings/*s")]
Settings,
```

and match it with:

```rust
MainRoute::SettingsHome | MainRoute::Settings => {}
```

This is still a less than ideal fix, but better than matching it in NotFound.

Reply:
Im like 80% sure SettingsHome wont match, let me try...

Reply:
Wait you wanted `SettingsHome` to be in `MainRoute` that is bad as we want all nested routes to be in the `SettingsRoute` enum.
Kind of defeats the purpose if to go home of settings i cant use `SettingsRoute` and am forced into using parent route.

Reply:
You can match `/settings/` again in the `SettingsRoute`?

```rust
MainRoute::SettingsHome | MainRoute::Settings => {
    html! { <Switch<SettingsRoute> render={Switch::render(switch_settings)} /> }
}
```

Reply:
ok that does work, let me update then...